test/swoosh/adapters/xml/helpers_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule Swoosh.Adapters.XML.HelpersTest do
+  use ExUnit.Case, async: true
+  alias Swoosh.Adapters.XML.Helpers, as: XMLHelper
+
+  setup do
```

Comment:
this can probably be a `setup_all`

Reply:
Was able to shift some of it out of the setup block. Had to keep the setup block around to run `Bypass.open` before every test and setup the host, but I was able to extract a majority of the config and the email generation out to the setup_all block.

Reply:
We were probably talking about different `setup` blocks here :) I'm so glad that you went and cleaned up the other block, though!

Reply:
@jdollar this `setup` block can definitely be changed for a `setup_all`

Reply:
@stevedomin @princemaple Yeah! You were right. I was thinking this was referring the setup block in the amazon SES test file originally. My bad. My selective reading kicked in or something. Update should be out there now.